bundles/org.openhab.binding.cp750/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,107 @@
+			<channel id="channel-mic" typeId="input-mic"/>
+			<channel id="channel-version" typeId="version"/>
+		</channels>
+
+		<config-description>
+			<parameter name="hostname" type="text" required="true">
+				<context>network-address</context>
+				<label>Hostname</label>
+				<description>Hostname or IP address of the device</description>
+			</parameter>
+			<parameter name="port" type="integer" min="1" max="65535">
+				<label>Port</label>
+				<default>61408</default>
+				<description>Port to access the device</description>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" unit="s" min="0">
+				<label>Refresh Interval</label>
+				<description>Interval the device is polled in sec.</description>
+				<default>5</default>
+			</parameter>
+			<parameter name="reconnectInterval" type="integer" unit="s" min="0">
+				<advanced>true</advanced>
+				<label>Retry interval</label>
+				<description>Reconnect interval in seconds after connection failure (0 = disabled)</description>
+				<default>10</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<!-- Sample Channel Type -->
+	<channel-type id="fader">
+		<item-type>Dimmer</item-type>
+		<label>Fader channel</label>
+		<description>Fader state</description>
+	</channel-type>
+	<channel-type id="mute">
+		<item-type>Switch</item-type>
+		<label>Mute channel</label>
+		<description>Mute state</description>
+	</channel-type>
+	<channel-type id="input">
+		<item-type>String</item-type>
+		<label>Current Input</label>
+		<description>Handles the state of INPUT_MODE</description>
+	</channel-type>
+	<channel-type id="input-analog">
+		<item-type>Switch</item-type>
+		<label>Analog Input Switch</label>
+		<description>Handles the state of INPUT_MODE=analog</description>
+	</channel-type>
+	<channel-type id="input-dig1">
+		<item-type>Switch</item-type>
+		<label>Digital 1 Input Switch</label>
+		<description>Handles the state of INPUT_MODE=dig-1</description>
+	</channel-type>
+	<channel-type id="input-dig2">
+		<item-type>Switch</item-type>
+		<label>Digital 2 Input Switch</label>
+		<description>Handles the state of INPUT_MODE=dig-3</description>
+	</channel-type>
+	<channel-type id="input-dig3">
+		<item-type>Switch</item-type>
+		<label>Digital 3 Input Switch</label>
+		<description>Handles the state of INPUT_MODE=dig-3</description>
+	</channel-type>
+	<channel-type id="input-dig4">
+		<item-type>Switch</item-type>
+		<label>Digital 4 Input Switch</label>
+		<description>Handles the state of INPUT_MODE=dig-4</description>
+	</channel-type>
+	<channel-type id="input-non_sync">
+		<item-type>Switch</item-type>
+		<label>Non-Sync Input Switch</label>
+		<description>Handles the state of INPUT_MODE=non_sync</description>
+	</channel-type>
+	<channel-type id="input-mic">
+		<item-type>Switch</item-type>
+		<label>Mic Input Switch</label>
+		<description>Handles the state of INPUT_MODE=mic</description>
+	</channel-type>
+	<channel-type id="version">
```

Comment:
Is there a particalr reason why the OS-version is modelled as channel and not as property of the thing?

Reply:
Moved this into a property